src/coreclr/jit/emitriscv64.cpp

```diff
@@ -1522,7 +1522,7 @@ void emitter::emitLoadImmediate(emitAttr size, regNumber reg, ssize_t imm)
             if (isSubtractMode)
             {
                 ins[numberOfInstructions - 1]    = INS_addi;
-                values[numberOfInstructions - 1] = -chunk;
+                values[numberOfInstructions - 1] = -(int32_t)chunk;
```

Comment:
This was a warning about unary minus on an unsigned variable leaves it unsigned.